operator/flags.go

```diff
@@ -253,7 +253,7 @@ func init() {
 	flags.String(option.K8sKubeConfigPath, "", "Absolute path of the kubernetes kubeconfig file")
 	option.BindEnv(option.K8sKubeConfigPath)
 
-	flags.Duration(operatorOption.NodesGCInterval, 0*time.Second, "GC interval for CiliumNodes")
+	flags.Duration(operatorOption.NodesGCInterval, 5*time.Minute, "GC interval for CiliumNodes")
```

Comment:
I might miss something here, can you explain why the default value is updated here. Node GC Interval is disabled if the value is 0s.

Reply:
I was following @kaworu's patch https://github.com/cilium/cilium/pull/19576#pullrequestreview-956454714, because we have a mismatch between default CLI value (0s) and helm value (5m), we should either change default CLI from 0s to 5m, or change default helm value from 5m to 0s to avoid confusion.

Reply:
oh right, thanks :100: